src/components/Registration/Registration.jsx

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './Registration.module.css';
+
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+
+const Registration = () => {
+  const submitRegister = (event) => {
+    axios
+      .post('http://localhost:4000/register', {
+        name: event.target.elements.name.value,
+        email: event.target.elements.email.value,
+        password: event.target.elements.password.value,
+      })
+      .then((response) => {
```

Comment:
Well, I think you can easily use `async/await` here instead of promise chains, you can use [this](https://stackoverflow.com/a/42964310) example to see the syntax for AF